### ISSUE TITLE
node: configuration for hare3 in standalone mode

### ISSUE THREAD

**Issue by dshulyak** [spacemeshos/go-spacemesh#5551]
standalone mode launches all spacemesh components in the same process.
it is currently missing timings for hare3 config and needs to be fixed

**Comment by dshulyak** [comment]
turns out that hare3 configuration is fine, the problem is with publish atx, see the following logs:

```
2024-02-14T10:47:03.648+0100	WARN	atxHandler	failed to process atx gossip	{"requestId": "c629809e", "sender": "12D3KooWPYs6EESRKFy8aS9isQhPPpEzUXuKDoBD3QSYvS3z3QCp", "errmsg": "fetch referenced atxs (prev 0000000000 pos fab6a97698 commit ccfe189fad): failed to fetch 1 hashes out of 1", "name": "atxHandler"}
2024-02-14T10:47:03.648+0100	INFO	atxBuilder	atx published	{"atx_id": "b8c6bd26f6", "challenge": "b056e83b3c", "smesher": "75fd1bbd44b991dd1723416c697207e8bc4120dc0ed5ef1c58d856d1e2957b38", "prev_atx_id": "0000000000", "pos_atx_id": "fab6a97698", "commitment_atx_id": "ccfe189fad", "vrf_nonce": 7, "coinbase": "standalone1qqqqqqp3qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqvqtwcg", "epoch": 2, "num_units": 1, "sequence_number": 0, "size": 310}
2024-02-14T10:47:06.906+0100	WARN	proposalBuilder	failed to build proposal	{"sessionId": "6bfffb25", "lid": 21, "errmsg": "empty active set", "name": "proposalBuilder"}
```

note that `atx published` is a lie, because libp2p simply discarded message, as it sees that it tried to process it and failed. i described it before https://github.com/spacemeshos/go-spacemesh/issues/4642

@pigmej

**Comment by dshulyak** [comment]
how to reproduce:

1. compile go-spacemesh from develop
2. launch it ./build/go-spacemesh --preset=standalone --genesis-time=2024-02-14T09:45:00.000Z -d /tmp/standalone


note that genesis time is set to be slightly into the future and has to be set in zulu timezone
